Hamming_Numbers/readme.md

```diff
@@ -0,0 +1,11 @@
+# Hamming Number
```

Comment:
Hey, can you add some more content in the README? Algorithm, pseudocode, example, time complexity etc?